api/nakadi-event-bus-api.yaml

```diff
@@ -0,0 +1,1207 @@
+swagger: '2.0'
+info:
+  title: Nakadi Event Bus API Definition
+  description: |
+    -----------
+    Definitions
+    -----------
+    Nakadi at its core aims at being a generic and content-agnostic event broker with a convenient API.
+    In doing this, Nakadi abstracts away, as much as possible, details of the backing messaging
+    infrastructure. The single currently supported messaging infrastructure is Kafka (Kinesis is planned
+    for the future).
+
+    In Nakadi every Event has an EventType, and a **stream** of Events is exposed for each registered
```

Comment:
@igoryankov we want to abstract and hide the idea of Kafka topic actually. How the streams are implemented, using kafka topics or some partitions in a special kafka topic is an implementation detail. 

My idea for the future would be that nakadi could automatically detect highly loaded event-types and repartition them automatically if needed, or move them to "slow" topics mixing with other slow event-types.